data/i18n/zh-cn/zh-cn.toml

```diff
@@ -218,6 +249,9 @@ other = "选项"
 [outdated_blog__message]
 other = "这篇文章已经一年多了，较旧的文章可能包含过时的内容。请检查从发表以来，页面中的信息是否变得不正确。"
 
+[post_create_issue]
+other = "登记一个问题"
```

Comment:
It's duplicated with upstream, I think we should remove it from `en.toml` and use Docsy version.

https://github.com/google/docsy/blob/main/i18n/zh-cn.toml#L44